src/main/js/siblings.js

```diff
@@ -28,37 +28,54 @@
  * OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-/*globals $:false, document:false, window:false */
+document.addEventListener('DOMContentReady', function () {
+  var DELTA = 600;
 
-$(document).ready(
-  function () {
-    "use strict";
-    $(window).scroll(
-      function () {
-        var $box = $('#talks'), more = $box.attr('data-more'), $tail = $('#tail');
-        if ($(window).scrollTop() >= $(document).height() - $(window).height() - 600 && more) {
-          $box.removeAttr('data-more');
-          $tail.show();
-          $.ajax(
-            {
-              url: more,
-              cache: false,
-              dataType: 'xml',
-              headers: { 'Accept': 'text/html' },
-              method: 'GET',
-              success: function (data) {
-                var $div = $(data).find('#talks');
-                $box.html($box.html() + $div.html());
-                $box.attr('data-more', $div.attr('data-more'));
-                $tail.hide();
-              },
-              error: function () {
-                $tail.html('Oops, an error :( Please, try to reload the page');
-              }
-            }
-          );
+  function needMore () {
+    return document.scrollingElement.scrollTop >=
+      document.body.offsetHeight - window.visualViewport.height - DELTA;
+  }
+
+  async function changeHandler () {
+    var $box = document.getElementById('talks');
+    var $tail = document.getElementById('tail');
+    var more = $box.dataset.more;
+
+    if (!more) {
+      return true;
+    }
+
+    if (needMore()) {
+      delete $box.dataset.more;
+      $tail.style.display = 'block';
+
+      var data = await fetch(more, {
+        method: 'GET',
+        cache: 'no-cache',
+        headers: {
+          'Accept': 'text/html',
+          'Content-Type': 'text/xml',
+        },
+      })
+        .then(function (res) { return res.text(); })
+        .catch(function (error) {
+          $tail.innerText = 'Oops, an error :( Please, try to reload the page';
+        });
+
+      if (data) {
+        var parser = new DOMParser();
+        var $fragment = parser.parseFromString(data, 'text/xml');
+        var $newBox = fragment.getElementById('talks');
+        var newEls = [...$newBox.children];
+        for (var idx = 0; idx < newEls.length; idx += 1) {
+          $box.appendChild(newEls[idx]);
         }
+        $box.dataset.more = $newBox.dataset.more;
+        $tail.style.display = 'none';
```

Comment:
Codacy has a fix for the issue: [Replace `'none'` with `"none"`](https://app.codacy.com/gh/yegor256/rultor/pullRequest?prid=9694263)
```suggestion
        $tail.style.display = "none";
```